akka-http-core/src/test/scala/akka/http/scaladsl/ClientServerSpec.scala

```diff
@@ -82,8 +82,8 @@ class ClientServerSpec extends AkkaSpecWithMaterializer(
     }
 
     "report failure if bind fails" in EventFilter[BindException](occurrences = 2).intercept {
-      val (hostname, port) = SocketUtil.temporaryServerHostnameAndPort()
-      val binding = Http().newServerAt(hostname, port).connectionSource()
+      val port = 1025 // non-root users typically can't bind to port numbers below 1024
```

Comment:
hmm, so this test will no longer succeed when something is bound to port 1025... *might* be OK?

Reply:
I think it should be okay because even if we pick any random port, there might be a possibility that something is bound to that random port?

Reply:
I agree picking a random port won't help. Using `temporaryServerHostnameAndPort` is a little better because it'll pick a port that at least was free right before the binding, but still leaves a race condition. Using port '0' for the first bind, and then binding the second materialization to that same port would be the most reliable solution, but also a somewhat more complicated refactor. As said, I guess perhaps using 1025 is good enough.